src/nft/components/explore/Banner.tsx

```diff
@@ -61,14 +61,22 @@ const HeaderContainer = styled.div`
 `
 const DEFAULT_TRENDING_COLLECTION_QUERY_AMOUNT = 5
 
+// Exclude collections that are not available in any of the following - OpenSea, X2Y2 and LooksRare:
+// CryptoPunkts
+const EXCLUDED_COLLECTIONS = ['0xb47e3cd837ddf8e4c57f05d70ab865de6e193bbb']
```

Comment:
We could also make this more dynamic where instead of a blocklist we filter collections where total number of  listings across the 3 marketplaces is 0. We can grab top 25, filter, and slice for max 5

Reply:
Yes! I also thought about that, but the ticket itself treats this as a one-off edge case and asks to filter it out. We can follow-up there too.

Reply:
agree with charlie--don't think we should introduce an exceptions list.

Reply:
Cool, I just let Patric know that we're changing this to be more generic. I will ping you when it's implemented.